website/sidebarsApi.js

```diff
@@ -134,6 +134,11 @@ export default {
           label: "Remote Proving",
           id: "generating-proofs/remote-proving",
         },
+        {
+          type: "doc",
+          label: "Remote Proving",
+          id: "generating-proofs/deterministic-builds",
+        },
```

Comment:
Unclear if this is the right place to place it overall.

Reply:
I think this might be better under "writing your zkvm application" but it's probably fine here too. We also talked about creating a separate "going to production" page and that seems like a better fit but maybe that could come later.

Reply:
/shrug I agree that this would fit better in a "going to production" section if such a section were to exist, but in the meantime seems fine to land it either in `writing your zkVM application` or in `generating proofs`.

Reply:
happy to kickoff that new section with this page & PR, and follow up on `next` with an overview of going to prod & a checklist of things to consider (audits, don't ever use / compile out dev mode, etc.)